public/scripts/Table.js

```diff
@@ -17,9 +39,17 @@ var TableHeader = React.createClass({
     render: function () {
         return (
             <tr>
-                {this.props.head.map(function (data, idx) {
-                    return <HeaderCell data={data}/>;
-                })}
+                {
+
+                    for(var p in obj1)
```

Comment:
`obj1` is the data that needs to be passed to the TableHeader React Class......
If so, it must be made available through props......

the error u must be getting is `undefined` one, ryt?...